PyRDF/Proxy.py

```diff
@@ -62,16 +88,77 @@ def GetValue(self):
         Value of the current action node
             This is the value obtained after executing the
             current action node in the computational graph.
-
         """
-        if not self.action_node.value:  # If event-loop not triggered
-            from . import current_backend
-            generator = CallableGenerator(self.action_node.get_head())
+        from . import current_backend
+        if not self.proxied_node.value:  # If event-loop not triggered
+            generator = CallableGenerator(self.proxied_node.get_head())
             current_backend.execute(generator)
 
-        return self.action_node.value
+        return self.proxied_node.value
 
     def _call_handler(self, *args, **kwargs):
         # Handles an operation call to the current action node
         # and returns result of the current action node.
         return getattr(self.GetValue(), self._cur_attr)(*args, **kwargs)
+
+
+class TransformationProxy(Proxy):
+    """
+    A proxy object to an instantiated node. Used as a controller of the user
```

Comment:
Here I would be more specific about what `TransformationProxy` does different with respect to `Proxy`. For example: 

```
A proxy object to non-action nodes. It implements access to the 
proxied_node's attributes and the creation of new graph nodes.
```

Reply:
done